xptifw/unit_test/xpti_api_tests.cpp

```diff
@@ -306,11 +309,16 @@ TEST(xptiApiTest, xptiNotifySubscribersGoodInput) {
   xptiForceSetTraceEnabled(true);
   int foo_return = 0;
   auto Result = xptiRegisterCallback(StreamID, 1, trace_point_callback2);
+  EXPECT_EQ(Result, xpti::result_t::XPTI_RESULT_SUCCESS);
   Result = xptiNotifySubscribers(StreamID, 1, nullptr, Event, 0,
                                  (void *)(&foo_return));
```

Comment:
It appears that the use of foo_return isn't tested here anymore. Is that intended?

Reply:
Yes, that's an input parameter, it is not changed anywhere. There's a lot of strange C-style casts throughout the codebase, which I would like to fix one day, but I'd prefer not to do it in this patch. I will fix it in this particular case though.